Resources/Prototypes/Roles/Jobs/Engineering/atmospheric_technician.yml

```
@@ -16,6 +16,9 @@
   - Engineering
   - External
   - Atmospherics
+  special:
+  - !type:AddImplantSpecial
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)